### PR TITLE
Add Rust sdk examples to api documentation

### PR DESCRIPTION
**Pull request by nkzou**
https://datadoghq.atlassian.net/browse/APITL-945

### What does this PR do? What is the motivation?

adds Rust SDK code examples to the public api docs

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

File: content/en/api/latest/_index.md
Comment (drichards-87) [inline content/en/api/latest/_index.md:205]:
```suggestion
Run `cargo add datadog-api-client`, or add the following to `Cargo.toml` under `[dependencies]`:
```

**Review by drichards-87 — CHANGES_REQUESTED** [review]

**Review by renteria1010 — APPROVED** [review]

**Review by drichards-87 — APPROVED** [review]
@nkzou Are you waiting for additional approvals, or are we good to merge this now?

**Comment by nkzou** [comment]
> @nkzou Are you waiting for additional approvals, or are we good to merge this now?

I'm planning to merge this in 2 weeks after our next set of Client library releases - currently this PR works because in `data/sdk_versions.json`, the rust sdk is set to the latest master branch and contains properly formatted examples. If we merge it now I think the bump sdk job will try to replace `master` with `0.1.3` which is an older version that will break the examples.

**Comment by drichards-87** [comment]
@nkzou Sounds good. You can reach out to me or the Documentation Slack channel when this is ready to be merged.

**Pull request merged**
